core/tests/protractor_desktop/extensions.js

```diff
@@ -146,8 +146,9 @@ describe('Interactions', function() {
         var confirmDeleteResponseButton = element(by.css(
           '.protractor-test-confirm-delete-response'));
         if (await deleteResponseButton.isPresent()) {
```

Comment:
I think we still need to figure out a good way to do this while allowing elements to load. We've tried doing it with `try ... except` blocks, but that doesn't seem to always work the way we'd like. Maybe we could specify our own failure handler for when the promise rejects?